PROTO_tests/tests/setops_test.py

```diff
@@ -680,30 +680,52 @@ def test_index_of(self):
         # index of nan (reproducer from #3009)
         s = ak.Series(ak.array([1, 2, 3]), index=ak.array([1, 2, np.nan]))
         assert ak.indexof1d(ak.array([np.nan]), s.index.values).to_list() == [2]
+        rng = np.random.default_rng()
+        seeds = [rng.choice(2**63), rng.choice(2**63), rng.choice(2**63), rng.choice(2**63)]
+        print("seeds: \n", seeds)
```

Comment:
Did you leave this print statement on purpose?  Same with the ones below?

Reply:
yep! I mentioned in the PR description that I modified the test to write out seeds to make future failures easier to reproduce